README.md

```diff
@@ -38,14 +39,55 @@ Then commit the changes to `go.mod` and `go.sum`.
 Using the provider
 ----------------------
 
-TODO - Fill this in for provider
+The RedisCloud Terraform provider is distributed through the HashiCorp managed [Terraform Registry](https://registry.terraform.io) 
+and is discovered through the use of a `required_providers`hcl block as shown in the following example.
+
+```hcl-terraform
+terraform {
+  required_providers {
+    rediscloud = {
+      source = "registry.redislabs.com/redislabs/rediscloud"
```

Comment:
does it need `registry.redislabs.com` at the beginning? or can it be just `redislabs/rediscloud` for general users?

Reply:
Just read your note on the PR :) 
I guess we can put this and the other comments all in the `developing the provider` section.

Reply:
Now my understanding is we can drop the `registry.redislabs.com` and use `redislabs/rediscloud`.  I can update it to be the expected value.  I mention this in the note.

Reply:
I have reworked the flow and this example.